### PR TITLE
file path fixes for osu redirects

### PR DESCRIPTION
**Pull request by cmyui**
# Describe your changes

## Checklist
- [x] I've manually tested my code

### CODE REVIEW

File: app/commands.py [hunk 1]
```diff
@@ -353,7 +353,7 @@ async def recent(ctx: Context) -> str | None:
 
 
 TOP_SCORE_FMTSTR = (
-    "{idx}. ({pp:.2f}pp) [https://osu.{domain}/beatmapsets/{map_set_id}/{map_id} "
+    "{idx}. ({pp:.2f}pp) [https://osu.{domain}/b/{map_id} "
```
Comment (cmyui) [inline app/commands.py:356]:
```suggestion
    "{idx}. ({pp:.2f}pp) [https://osu.{domain}/beatmapsets/{map_set_id}/{map_id} "
```

File: app/objects/beatmap.py [hunk 2]
```diff
@@ -230,7 +230,7 @@ class Beatmap:
 
     Properties:
       Beatmap.full -> str # Artist - Title [Version]
-      Beatmap.url -> str # https://osu.cmyui.xyz/beatmapsets/123/321
+      Beatmap.url -> str # https://osu.cmyui.xyz/b/321
```
Comment (cmyui) [inline app/objects/beatmap.py:233]:
```suggestion
      Beatmap.url -> str # https://osu.cmyui.xyz/beatmapsets/123/321
```

File: app/api/domains/cho.py [hunk 3]
```diff
@@ -1539,7 +1538,9 @@ async def handle(self, player: Player) -> None:
         elif player.match.map_id == -1:
             if player.match.prev_map_id != self.match_data.map_id:
                 # new map has been chosen, send to match chat.
-                map_url = f"https://osu.{app.settings.DOMAIN}/beatmapsets/#/{self.match_data.map_id}"
+                map_url = (
+                    f"https://osu.{app.settings.DOMAIN}/b/{self.match_data.map_id}"
+                )
```
Comment (cmyui) [inline app/api/domains/cho.py:1543]:
```suggestion
                map_url = f"https://osu.{app.settings.DOMAIN}/beatmapsets/#/{self.match_data.map_id}"
```

**Review by tsunyoku — CHANGES_REQUESTED** [review]
you had one job 😭

/b won't be clickable to a map ingame, it will open in browser even if you have it

there was another pr related to the urls where i said to construct the whole valid url, i'm going to say the same here.

**Comment by cmyui** [comment]
> you had one job 😭
> 
> /b won't be clickable to a map ingame, it will open in browser even if you have it
> 
> there was another pr related to the urls where i said to construct the whole valid url, i'm going to say the same here.

Was this feedback meant for the other PR? (this PR doesn't do anything related to /b/)

**Comment by tsunyoku** [comment]
> > you had one job 😭
> 
> > 
> 
> > /b won't be clickable to a map ingame, it will open in browser even if you have it
> 
> > 
> 
> > there was another pr related to the urls where i said to construct the whole valid url, i'm going to say the same here.
> 
> 
> 
> Was this feedback meant for the other PR? (this PR doesn't do anything related to /b/)

diff was showing some beatmap url changes but they're gone now

**Review by tsunyoku — APPROVED** [review]

**Comment by NiceAesth** [comment]
yell if it breaks lgtm

**Pull request merged**